packages/core/src/bind/connectFactoryObservable.ts

```diff
@@ -68,19 +68,23 @@ export default function connectFactoryObservable<A extends [], O>(
 
       return source$.subscribe(subscriber)
     }) as BehaviorObservable<O>
-    const reactObservable$ = reactEnhancer(publicShared$)
+    publicShared$.getValue = getEmpty
```

Comment:
Why is it `getEmpty` by default instead of `sharedObservable$.getValue`? I think if no subscription happened yet, and someone calls `reactGetValue`, shouldn't it directly poke `sharedObservable$`?

Reply:
sure... I mean, in that case `sharedObservable$.getValue will return `EMPTY_VALUE`, so it really is the same. But yeah, I'll make the change. Thanks!